Samples~/AvatarCreatorSamples/AvatarCreatorExperimental/Scripts/SimpleAvatarCreator.cs

```diff
@@ -137,7 +187,9 @@ private void SetupAvatar()
         {
             avatar.AddComponent<MouseRotationHandler>();
             avatar.AddComponent<AvatarRotator>();
-            avatar.GetComponent<Animator>().runtimeAnimatorController = animationController;
+            var animator = avatar.GetComponent<Animator>();
+            animator.runtimeAnimatorController = animationController;
```

Comment:
Minor thing but probably better to set the runtimeAnimatorController after calling the SetupAnimator function.